README.md

```diff
@@ -1 +1,112 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers (JS, PHP, Python, Ruby, Go, Java, .NET)
+- [ ] mysqldump
+- [ ] grafana
+- [ ] BI (Tableau, PowerBI, pick one)
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions with basic MySQL auth implementation
+- [ ] Audit logs (connections, queries)
+
+### Get web interfaces enterprise ready [APPS]
+
+- [ ] gitbase-web SSO (protocol TBD)
+- [ ] bblfsh-web SSO (protocol TBD)
+
+### Gitbase research [DR]
+
+- [ ] Support updates in backing data (indexes, etc)
+- [ ] Usage of persistent caches for UAST and others
+
+### Launch gitbase-spark-connector [DR]
+
+- [ ] Launch gitbase-spark-connector community beta version.
+- [ ] Launch gitbase-spark-connector enterprise beta version.
+
+### Release necessary language support for enterprise [LA]
+
+- [ ] C# driver
+- [ ] C/C++ driver
+
+### Get bblfsh ready for wide language support of our use cases [LA]
+
+- Finish migration to v2 for all supported client languages (JVM, Python, Go)
+- Deprecate v1
+- Fix positions in all alpha+ drivers
+
+### Have an available demo cluster [INFRA]
+
+- [ ] Deploy a demo cluster for 'Code as Data' that we can use for demos and 3rd parties
```

Comment:
I assume this is a separate cluster, not the pipeline cluster, isn't it?

Reply:
@rporres AFAIK this is hard requirement, but we may just use GKE for this.
cc @eiso

Reply:
It's not that I don't like it, just remembering conversations around this in the past

Reply:
This is something that I'd like to be able to use when demoing the engine with large customers.
Somewhere that is running some stable release of our tooling and with all the necessary set-up to make everything work as fast as possible.

Reply:
We have to specify then the needs, especially if we need things to be ready very fast as things do not go very fast in the baremetal world. Another thing to consider is that the minimum renting time of servers in Online is one month.